tests/python/pytest_core.py

```diff
@@ -52,6 +50,12 @@ def __repr__(self):
         return f"[SampleInput args={self.args} kwargs={self.kwargs}]"
 
     def jax(self):
+        from pytest_utils import JAX_AVAILABLE
+
+        if JAX_AVAILABLE:
```

Comment:
nitpick: this condition doesn't do anything here.

i.e. if jax is not available it's still going to fail when we try to use jnp/torch_to_jax_dtype_map below.
Maybe throw an error instead of guard the import here.